CHANGELOG.md

```diff
@@ -2,15 +2,17 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
-## 9.5.0-alpha.1 - September 3, 2020
-[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.4.0...android-v9.5.0-alpha.1) since [Mapbox Maps SDK for Android v9.4.0](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.4.0)
-### Changes made to match core library v4.0.0 updates
+## 9.5.0-beta.1 - September 14, 2020
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.5.0-beta.1...android-v9.4.0) since [Mapbox Maps SDK for Android 9.4.0](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.4.0)
```

Comment:
```suggestion
[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.4.0...android-v9.5.0-beta.1) since [Mapbox Maps SDK for Android 9.4.0](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.4.0)
```

Reply:
Fixed.